### ISSUE TITLE
Get comments in dxf files

### ISSUE THREAD

**Issue by christellenetwork** [mozman/ezdxf#103]
Hello,

Is there a reason that the comments (999) are ignored in the dxf files ?
I need them for a client because they contains important info I need to get, is there a way to get them ?

Thank you

**Comment by mozman** [comment]
There are no rules to bind comments to DXF structures, so I don't know where to store them. And there exist much better structures in DXF to store 3rd party application data, like XDATA for entities!

But it is not that hard to get comments from a text file in Python ;-).

**Comment by mozman** [comment]
Added a howto example for reading comments:  https://github.com/mozman/ezdxf/blob/master/examples/load_comments.py

This example also take into account the correct file encoding and checks if file is a DXF file.

**Comment by christellenetwork** [comment]
Hey, thak you for your responses and link.

I succeeded in overriding some files and classes but it was a bit hard (I am not an expert in Python).

The comments were needed when getting the structure of the dxf, each entity has its comment. Unfortunately, the dxf the client provides contains sometimes multiple "objects" on the same layer and the only way to separate the "objects" were the info in the comments...

I didn't want to open the dxf again to get the comments in a more traditionnal way because I would miss the link with each entity.

Nevertheless I managed to get what I needed :)

Thank you, have a nice a day
